MapboxNavigation/NavigationViewController.swift

```diff
@@ -200,6 +200,26 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
     
     var bottomViewController: ContainerViewController?
     
+    public typealias FloatingButtonsPositions = NavigationView.FloatingButtonsPositions
```

Comment:
Instead of declaring the enumeration as a nested type and exposing it at the top level via a type alias, declare the enumeration here at the top level, so no type alias is needed.

Reply:
Thanks for the suggestion, I'll make it public at a top level to avoid typealias and nested type.